data/alexandria start.txt

```diff
@@ -0,0 +1,1304 @@
+			label questions
+			`	"Can we leave, or will you ask more of your fekkik questions?"`
+			choice
+				`	"What did you read in the book we just captured?"`
+					goto book
+				`	"Will we have escorts this time?"`
+					goto escorts
+				`	"Will I get paid for the fifth book, even though we did not find it?"`
+					goto pay
+				`	"Let's go."`
+					goto accept
+				`	"Your insane quest is a waste of my time.  Go find another captain."`
+					decline
+			label escorts
+			`"No, my escort funding has run out.  If you take a safe route, we are unlikely to die by pirates.  Fly to Earth, then the Paradise Planets, and then west to the Deep."`
+				goto questions
+			label pay
+			`"As long as you pick up the book, you'll get paid."`
+				goto questions
+			label book
+			`"As you recall from the first two books, the Chal Jub Paqmey is an opera about a merchant, Hod Qav (of unspecified gender) destined to be the greatest warrior of the era.  The first book, the Paq wa'DIch, describes Hod Qav leaving their clan Botolphstone and wandering a continent that seems vast.  The second book, the Paq cha'DIch, describes the continent's politics descending into chaos and civil war.  Meanwhile, the merchant wanders distracted, fighting bandits, gathering weapons, and hoarding money.`
+			`	"In the third book, the Paq WejDIch, the merchant joins the war, fighting for the side they view as right.  Eventually, the merchant re-unites their continent and brings peace.`
+			`	"In the fourth book, the Paq LoSDIch, tells of Hod Qav finding the ship Supghor, so large it can cross oceans.  Hod Qav wanders the world discovering new civilizations that nobody from Botolphstone had even heard of.`
+			`	"Oddly, instructins in the Chal Jub Paqmey say the third and fourth books can be reversed when the opera is performed.  HoD Qav can explore the world and gather power, and then return to their continent to bring peace.  Or, the merchant can bring peace to their continent and then explore the world."`
+				goto questions
+			label accept
+			`"Finally."`
+				accept
+	on visit
+		dialog `You've forgotten your senile friend or his pencils.  Perhaps they're in one of your escorts?`
+	on complete
+		conversation
+			`A Deep Security officer runs towards you and Daemos in a hurry yelling, "it's been disabled!  We have to leave!"`
+			`	When the officer reaches you, he explains the Alexandria Seeker who was supposed to meet you is in grave trouble.  The Seeker's ship is disabled in an uninhabited system called Mora.'  The Seeker, Gonza Doorin, refuses to let anyone onto his ship except another Seeker.  There is a pirate fleet in the system, attacking anyone who gets near the ship."`
+			`	Daemos interjects, "can Deep Security escort us?"`
+			`	"Of course," the officer responds.  "We must leave at once."`
+			`	"You'll need to refuel at Tania Australis before heading to Mora," warns Daemos.  "Meet us in the spaceport as soon as you're ready to go."`
+
+
+
+government "Librarian"
```

Comment:
That would make the mission impossible.  The Star Queen would be destroyed in seconds.

Also, the library maintains peaceful relations with the pirates.  One of the Seeks I'm planning involves setting up a distribution network of illegal literature printed in anarchist planets with pirates paid to smuggle the books into civilized space.  Surely as an anarchist yourself, you would understand the appeal.

In the Alexandria Library Card mission, Daemos Titan is expected by the marauders and greeted as a business client rather than their next victim.  He says he "knows someone" that can forge a signature, implying he has done business with the marauders before.